content/tutorials/permissions.md

```diff
@@ -1,15 +1,15 @@
 ---
 title: Fine-Grained Permissions 
-description: Control who can publish, consume or browse data
+description: Control who can publish, download or browse data
```

Comment:
I see that in the previous document, you corrected the same sentence with "buy" instead of "consume". Should we do the same on this file?